tests/sentry/api/endpoints/test_group_tags.py

```diff
@@ -1,102 +0,0 @@
-from __future__ import absolute_import
-
-from sentry import tagstore
-from sentry.testutils import APITestCase
-
-
-class GroupTagsTest(APITestCase):
```

Comment:
Why was this removed instead of replacing tag creation with event store  to produce the same result ?

Reply:
Sorry, I did not delete it but renamed the file earlier. Reverted that change now.

Reply:
ok